src/ucs/config/parser.c

```diff
@@ -331,6 +331,17 @@ void ucs_config_help_enum(char *buf, size_t max, const void *arg)
     __print_table_values(arg, buf, max);
 }
 
+ucs_status_t ucs_config_clone_comp(const void *src, void *dst, const void *arg)
+{
+    const ucs_log_component_config_t *src_comp  = src;
```

Comment:
pls, 1 excessive whitespace can be removed

Reply:
done